src/components/authorization/dto/powers.ts

```diff
@@ -7,6 +7,7 @@ export enum Powers {
   CreateBudget = 'CreateBudget',
   CreateBudgetRecord = 'CreateBudgetRecord',
   CreateCeremony = 'CreateCeremony',
+  CreateChangeRequest = 'CreateChangeRequest',
```

Comment:
This isn't necessary as it's not a root object. Instead we would want to use `project.changeRequests.canCreate`

Reply:
Make sure that the roles are updated accordingly for `project.changeRequests.write = true`

Reply:
Deferring this to a later PR